client/src/modules/profiles/pages/userProfile.tsx

```diff
@@ -94,9 +94,34 @@ export const UserProfilePage = () => {
             loadingText={'Saving Profile Changes'}
             submitText={'Save Profile Changes'}
           />
-          <Button colorScheme={'red'} marginBlock={'2em'} onClick={clickDelete}>
-            Delete My Data
-          </Button>
+          <Flex gap="2em" marginBlock={'2em'}>
+            <Button
+              height={'2.8em'}
+              paddingBlock={'.65em'}
+              paddingInline={'.4em'}
+              colorScheme={'red'}
+              onClick={clickDelete}
+            >
+              Delete My Data
+            </Button>
+            <Link
+              fontWeight="600"
+              background={'gray.85'}
+              color={'gray.10'}
+              height={'100%'}
+              size={'lg'}
+              borderRadius={'5px'}
+              paddingBlock={'.65em'}
+              paddingInline={'.4em'}
+              _hover={{ color: 'gray.85', backgroundColor: 'gray.10' }}
+              download={`${userInfo.name}.json`}
+              href={`data:text/json;charset=utf-8,${encodeURIComponent(
+                JSON.stringify(userInfo),
+              )}`}
```

Comment:
Since `userInfo` has noise like `__typename` in it, we'd be better off filtering that out.  We can do that via

```ts
const createDownloadData = (userInfo: UserProfileQuery['userInformation']) => {
  const dataString = JSON.stringify(userInfo, (key, value) =>
    key === '__typename' ? undefined : value,
  );
  return `data:text/json;charset=utf-8,${encodeURIComponent(dataString)}`;
};
```

which can go outside the component, since it's a pure function.